luxonis_ml/data/loaders/luxonis_loader.py

```diff
@@ -212,8 +212,8 @@ def _load_image_with_annotations(self, idx: int) -> Tuple[np.ndarray, Labels]:
         if not self.dataset.is_remote:
             img_path = ann_rows[0][8]
         elif not self.stream:
-            uuid = ann_rows[0][0]
-            img_path = next(self.dataset.media_path.glob(f"{uuid}.*"))
+            uuid, file_extension = ann_rows[0][0], ann_rows[0][8].rsplit(".", 1)[-1]
```

Comment:
Nitpick: Can we split this into two lines for better code readability?

Reply:
Okay, I’ve done it.